### PR TITLE
Validation: The filename and it's id in the file should be in sync

### PR DESCRIPTION
**Pull request by martin-fleck-at**
- Add warning if filename and element id do not match
- Provide quick fix to properly rename file

Minor:
- Fix watch scripts for browser and electron to use package name
- Fix wrong translation of diagnostic severity

### CODE REVIEW

**Review by harmen-xb — COMMENTED** [review]
Code looks good, some minor changes requested.

File: extensions/crossmodel-lang/src/language-server/cross-model-code-action-provider.ts [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+      };
+      for (const diagnostic of params.context.diagnostics) {
+         if (FilenameNotMatchingDiagnostic.is(diagnostic)) {
+            accept(this.fixFilenameNotMatching(diagnostic, document));
+         }
+      }
+      return result;
+   }
+
+   private fixFilenameNotMatching(diagnostic: FilenameNotMatchingDiagnostic, document: LangiumDocument): CodeAction | undefined {
```
[the remark this answers is not in the document]
Comment (martin-fleck-at) [reply extensions/crossmodel-lang/src/language-server/cross-model-code-action-provider.ts:28]: Good catch! I'll properly mark the action as quickfix!

File: extensions/crossmodel-lang/src/language-server/cross-model-validator.ts [hunk 2]
```diff
@@ -56,6 +73,27 @@ export class CrossModelValidator {
    checkNode(node: AstNode, accept: ValidationAcceptor): void {
       this.checkUniqueGlobalId(node, accept);
       this.checkUniqueNodeId(node, accept);
+      this.checkMatchingFilename(node, accept);
+   }
+
+   protected checkMatchingFilename(node: AstNode, accept: ValidationAcceptor): void {
+      if (!isEntity(node) && !isMapping(node) && !isRelationship(node)) {
```
[the remark this answers is not in the document]
Comment (martin-fleck-at) [reply extensions/crossmodel-lang/src/language-server/cross-model-validator.ts]: Diagrams only have an optional `id` but I added the check that iff the id is set, it actually needs to match.

**Comment by martin-fleck-at** [comment]
@harmen-xb I pushed an update that should address all your concerns and also fixed the test issues.

**Review by harmen-xb — APPROVED** [review]
Changed look good, it's also working nicely now with the quick fix.

**Pull request merged**